server/tests/customer.test.ts

```diff
@@ -13,6 +13,9 @@ import {
 } from './testUtils';
 chai.use(chaiHttp);
 
+const byId = <T extends { id: number } | { id: string }>(a: T, b: T) =>
+  a.toString().localeCompare(b.toString());
```

Comment:
I may not be interpreting this correctly, but shouldn't this be `a.id.toString().localeCompare(b.id.toString());` if we are comparing ids?

Reply:
yes it should, good catch 👍